### PR TITLE
RFC: 0107 added checklist instructions

### PR DESCRIPTION
**Pull request by iliapolo**
Added instructions on creating the graduation issue task list.

[Rendered version](https://github.com/aws/aws-cdk-rfcs/blob/epolon/graduation-issue-checklist/text/0107-construct-library-module-lifecycle.md)

---

_By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache-2.0 license_

### CODE REVIEW

File: text/0107-construct-library-module-lifecycle.md [hunk 1]
```diff
@@ -202,14 +202,22 @@ The module is generally available with a backwards compatible guarantee across m
...
 - **Create Graduation Issue** - Create a GitHub issue titled `Graduate @aws-cdk/{module} to stable`.
 This issue communicates our intent to graduate this module. It will also serve as an asynchronous channel for anyone who wants to provide feedback.
 Update the module tracking issue with a link to this issue so that the community can be notified.
 
   > External Stakeholders and major community contributors should be contacted directly.
 
+  Include a checklist of tasks that must be completed before flipping the switch.
+  Initially, this would be:
+
+  - **Validate existing bugs** - Make sure there weer no *p0* bugs in the last 4 weeks.
```
Comment (nija-at) [inline text/0107-construct-library-module-lifecycle.md]: typo: 'where'

File: text/0107-construct-library-module-lifecycle.md [hunk 2]
```diff
@@ -202,14 +202,22 @@ The module is generally available with a backwards compatible guarantee across m
...
+  - **Validate L2 coverage** - Consult module owners and make sure we have sufficient L2 coverage to provide a holistic developer experience.
```
Comment (nija-at) [inline text/0107-construct-library-module-lifecycle.md]: Provide a little more detail into what 'sufficient' means. I would say something along the lines of, 'must be possible to configure the service for most major use cases via the CDK'

File: text/0107-construct-library-module-lifecycle.md (hunk 1 above)
Comment (iliapolo) [reply text/0107-construct-library-module-lifecycle.md]: 👍

File: text/0107-construct-library-module-lifecycle.md (hunk 2 above)
Comment (iliapolo) [reply text/0107-construct-library-module-lifecycle.md]: 👍

File: text/0107-construct-library-module-lifecycle.md [hunk 3]
```diff
@@ -202,14 +202,23 @@ The module is generally available with a backwards compatible guarantee across m
...
 Update the module tracking issue with a link to this issue so that the community can be notified.
 
   > External Stakeholders and major community contributors should be contacted directly.
 
+  Include a checklist of tasks that must be completed before flipping the switch.
+  Initially, this would be:
+
+  - **Validate existing bugs** - Make sure there were no *p0* bugs in the last 4 weeks.
+  - **Validate L2 coverage** - Consult module owners and make sure we have sufficient L2 coverage to provide a holistic developer experience,
+  i.e most major use cases can be configured via CDK, without needing prior manual setup.
```
Comment (nija-at) [inline text/0107-construct-library-module-lifecycle.md]: Drop the 'prior manual step' addendum or mention it as an example.

This is just one example of where a use case is not completely covered.

**Pull request merged**